### PR TITLE
Ledger doc

### PR DESCRIPTION
**Pull request by achamayou**
Doc for #1135

### CODE REVIEW

File: doc/operators/ledger.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+It is important to note that while all entries stored in files ending in ``.committed`` are committed, not all committed entries
+are stored in such a file at any given time. A number of them are typically in the in-progress files, waiting to be flushed to
+a ``.committed`` file once the size threshold is met.
+
+The listing below is an example of what a ledger directory may look like.
+
+.. code-block:: bash
+
+    -rw-rw-r-- 1 user user 1.6M Jun 16 14:08 ledger_1-7501.committed
```
Comment (jumaffre) [inline doc/operators/ledger.rst:34]: It'd be useful to show the command that displayed those and how it relates to the `--ledger-dir` `cchost` parameter.

File: doc/operators/ledger.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
+Ledger
```
Comment (jumaffre) [inline doc/operators/ledger.rst:1]: There's already a `Ledger` page under `Developers/` (https://microsoft.github.io/CCF/master/developers/ledger.html), might be worth merging those two.

File: doc/operators/ledger.rst [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+Location
+--------
+
+Nodes can be configured to store their ledger under a particular directory with the `--ledger-dir` command-line option.
+
+File layout
+-----------
+
+The ledger directory contains a series of files. File size is controlled by the ``--ledger-chunk-max-bytes`` command line switch.
```
Comment (jumaffre) [inline doc/operators/ledger.rst:18]: s/switch/argument as "switch" sounds like a flag?

File: doc/operators/ledger.rst (hunk 1 above)
Comment (achamayou) [reply doc/operators/ledger.rst:34]: will do

File: doc/operators/ledger.rst (hunk 2 above)
Comment (achamayou) [reply doc/operators/ledger.rst:1]: Yeah I know, that's where the release notes point to now. It's a bit different though, it explains how the ledger is formed, it's more of a design document than a reference op doc.

File: doc/operators/ledger.rst (hunk 3 above)
Comment (achamayou) [reply doc/operators/ledger.rst:18]: yeah, although I think "option" may be right here.

File: doc/operators/ledger.rst [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+Nodes can be configured to store their ledger under a particular directory with the `--ledger-dir` command-line option.
+
+File layout
+-----------
+
+The ledger directory contains a series of files. File size is controlled by the ``--ledger-chunk-max-bytes`` command line option.
+
+Files containing only committed entries are named ``ledger_$STARTSEQNO-$ENDSEQNO.committed``. These files are closed and immutable,
+it is safe to replicate them to backup storage. They are identical across nodes, provided ``--ledger-chunk-max-bytes`` has been set to the same value.
```
Comment (ashamis) [inline doc/operators/ledger.rst:21]: I would explicitly state that they cannot be deleted from local storage (for now)

File: doc/operators/ledger.rst [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+ 
+    $ ./cchost --ledger-dir $LEDGER_DIR ...
+    $ cd $LEDGER_DIR
+    $ ls
+    -rw-rw-r-- 1 user user 1.6M Jun 16 14:08 ledger_1-7501.committed
+    -rw-rw-r-- 1 user user 1.2M Jun 16 14:08 ledger_12502-17501.committed
+    -rw-rw-r-- 1 user user 1.1M Jun 16 14:08 ledger_17502-22501.committed
+    -rw-rw-r-- 1 user user 1.1M Jun 16 14:08 ledger_22502-27501.committed
+    -rw-rw-r-- 1 user user 1.1M Jun 16 14:08 ledger_27502-32501.committed
+    -rw-rw-r-- 1 user user 1.1M Jun 16 14:08 ledger_32502-37501.committed
```
Comment (ashamis) [inline doc/operators/ledger.rst:42]: you probably can have less than ~20 committed files

File: doc/operators/ledger.rst (hunk 4 above)
Comment (achamayou) [reply doc/operators/ledger.rst:21]: Yes, very good point, thank you.

File: doc/operators/ledger.rst (hunk 5 above)
Comment (achamayou) [reply doc/operators/ledger.rst:42]: Yeah, I'll elide the middle ones.

**Pull request merged**